controllers/dspipeline_params.go

```diff
@@ -548,16 +587,103 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
-		// If a Custom CA Bundle is specified for injection into DSP API Server Pod
-		// then retrieve the bundle to utilize during storage health check
+		// Check for Global certs
```

Comment:
no need to change this, but I find the name "global" lacking. What we really mean is "provided by the platform instead of by the DSPA user". I think I would have called them odh platform certs or something like that.

Reply:
updated!